### ISSUE TITLE
[Paper Note] Forecasting Downstream Performance of LLMs With Proxy Metrics, Arkil Patel+, arXiv'26, 2026.05

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#5573]
# URL
- https://arxiv.org/abs/2605.18607
# Authors
  - Arkil Patel
  - Siva Reddy
  - Marius Mosbach
  - Dzmitry Bahdanau
# Abstract
  - Progress in language model development is often driven by comparative decisions: which architecture to adopt, which pretraining corpus to use, or which training recipe to apply. Making these decisions well requires reliable performance forecasts, yet the two commonly used signals are fundamentally limited. Cross-entropy loss is poorly aligned with downstream capabilities, and direct downstream evaluation is expensive, sparse, and often uninformative at early training stages. Instead, we propose to construct proxy metrics by aggregating token-level statistics, such as entropy, top-k accuracy, and expert token rank, from a candidate model's next token distribution over expert-written solutions. Across three settings, our proxies consistently outperform loss- and compute-based baselines: 1) For cross-family model selection, they rank a heterogeneous population of reasoning models with mean Spearman Rho = 0.81 (vs. Rho = 0.36 for cross-entropy loss); 2) For pretraining data selection, they reliably rank 25 candidate corpora for a target model at roughly $10{,}000\times$ less compute than direct evaluation, pushing the Pareto frontier beyond existing methods; and 3) for training-time forecasting, they extrapolate downstream accuracy across an $18\times$ compute horizon with roughly half the error of existing alternatives. Together, these results suggest that expert trajectories are a broadly useful source of signal for assessing model capabilities, enabling reliable performance forecasting throughout the model development life cycle.
# Translation (by gpt-5-nano)
- 言語モデル開発の進展は、どのアーキテクチャを採用するか、どの事前学習コーパスを用いるか、どのトレーニングレシピを適用するかといった比較に基づく意思決定によって推進されることが多い。これらの意思決定を適切に行うには信頼性のある性能予測が必要だが、一般に用いられる二つの信号は本質的に限界を抱えている。クロスエントロピー損失は下流の能力と乖離しており、直接的な下流評価は高コストでスパース、かつ初期訓練段階では有益でないことが多い。代わりに、候補モデルの専門家が執筆した解答に対する次のトークン分布から、エントロピー、トップk精度、専門家トークン順位といったトークンレベルの統計を集約して代理指標を構築することを提案する。3つの設定において、我々の代理指標は損失ベースおよび計算量ベースのベースラインを一貫して上回る。1) ファミリ横断のモデル選択では、推論モデルの異種混在群を平均Spearmanρ=0.81でランク付けする（クロスエントロピー損失ではρ=0.36と比較）。2) 事前学習データの選択では、対象モデルに対して25件の候補コーパスを信頼性高く序列付けし、直接評価の約1/10,000の計算量で評価できるため、パレート前面を既存手法より超えて押し広げる。3) 訓練時の予測では、18倍の計算量の範囲にわたって下流の精度を外挿し、既存の代替手法のおおよそ半分の誤差である。これらを総合すると、専門家の軌跡はモデル能力を評価するための広く有用な信号源であることを示唆しており、モデル開発ライフサイクル全体を通じた信頼性の高い性能予測を可能にする。
# Summary (by gpt-4o-mini)
- 信頼性の高い性能予測が必要な言語モデル開発において、クロスエントロピー損失や直接評価には限界があることを指摘し、代わりに専門家が執筆した解答のトークン分布からエントロピーや精度といったトークンレベルの統計を用いた代理指標を提案。これにより、モデル選択や事前学習データの選択、訓練時の予測において一貫して優れた結果を示し、専門家の軌跡がモデル能力評価において有用な信号であることを明らかにした。

**Comment by AkihikoWatanabe** [comment]
元ポスト: https://x.com/rosinality/status/2056644552375632273?s=20

**Comment by AkihikoWatanabe** [comment]
著者ポスト: https://x.com/arkil_patel/status/2057819898349572156?s=20

**Comment by AkihikoWatanabe** [comment]
クロスエントロピーlossに代わるcandidate modelのdownstreamタスクの性能を間接的に測定するための代理指標の提案で、クロスエントロピーlossと比較。代理指標はexpertが作成したtrajectoryに対するcandidate modelのnext token predictionの分布（や、エントロピー等指標に基づく重みづけの組み合わせ）によって、算出される（式1, 2）。

<img width="1120" height="467" alt="Image" src="https://github.com/user-attachments/assets/ca380142-80ed-456c-8495-bef1216ff698" />

6つの異なるモデルファミリーの18種類のreasoning modelにおいて、6種類のベンチマークにおいて、モデルのdownstreamタスク性能をランク付けできるかをSpearman Rhoで測定したところ、クロスエントロピーlossが0.36だったのに対し、提案した代理指標（を特徴量として用いたRankSVM）は0.81を記録。また、（あるLLMがある事前学習コーパスで学習された場合のdownstreamタスクでの性能の良さによって）事前学習コーパスの良さをランク付けするタスクの場合、ベースラインと比較して10,000倍計算コストを削減できたとのこと。

<img width="1114" height="752" alt="Image" src="https://github.com/user-attachments/assets/48ad7e9e-0f78-41a5-ad47-9464df25a278" />

**Comment by AkihikoWatanabe** [comment]
DataDecide testbed:
- #5628
